stamina-testkit/src/test/scala/stamina/testkit/ScalatestTestGenerationSpec.scala

```diff
@@ -59,7 +59,7 @@ class ScalatestTestGenerationSpec extends StaminaTestKitSpec {
 
     "the sample is only for version 2 and up" should {
       val spec = new StaminaTestKit with WordSpecLike {
-        val persisters = Persisters(ItemPersister[V2]("item1"))
+        val persisters = Persisters(ItemPersister("item1"))
         "TestDomainSerialization" should {
           persisters.generateTestsFor(
             sample(item1).from[V2]
```

Comment:
Hmm. The reason you'd use `.from[V2]` would be when you're adding a sample that cannot be represented in `V1`.. so I suppose it doesn't make much sense to use `item1` both for the 'normal' sample and the 'only from V2' sample. I do think I'd like to keep the different tests self-contained